src/app/helptext/jails/jails-add.ts

```diff
@@ -480,8 +480,8 @@ hostid_tooltip: T('A new jail hostid, if desired. \
 <i>1a2bc345-678d-90e1-23fa-4b56c78901de</i>.'),
 
 hostid_strict_check_placeholder: T('hostid_strict_check'),
-hostid_strict_check_tooltip: T('Checks the hostid property of the jail \
-and if they are not the same as the hosts, it does not start the jail'),
+hostid_strict_check_tooltip: T('Check the <i>hostid</i> property of the \
+jail. If not the same as the hosts, do not start the jail'),
```

Comment:
"hosts" here is meant as a possessive, that can be fixed by just making it "host".